app/views/projects/_destroy.html.erb

```diff
@@ -1,17 +1,57 @@
 <%= viral_card do |card| %>
   <% card.with_header(title: t(:"projects.edit.advanced.destroy.title")) %>
   <% card.with_section do %>
+    <div class="hidden confirm-destroy">
+      <div
+        class="
+          flex p-4 mb-4 text-red-800 rounded-lg bg-red-50 dark:bg-slate-800
+          dark:text-red-400
+        "
+      >
+        <%= viral_icon(name: "exclamation_circle", classes: "h-6 w-6 flex-none") %>
+        <div class="flex-1 ml-4">
+          <%= t(
+            :"projects.edit.advanced.destroy.confirm.warning.pre_html",
+            project_name: @project.human_name,
+          ) %>
+          <ul class="py-2 ml-4 max-w-md space-y-1 list-disc list-inside">
+            <li><%= t(
+                :"projects.edit.advanced.destroy.confirm.warning.samples_count",
+                count: @samples_count,
+              ) %></li>
+            <li><%= t(
+                :"projects.edit.advanced.destroy.confirm.warning.files_count",
```

Comment:
We probably don't need to list the files count here as its implied by the samples_count above that its linked data will be removed

Reply:
Whoops I don't know what I was thinking when I typed that. I meant to say the project being deleted implies that any data(files) attached to it would be removed. So yeah I think we should still remove it